File: particle-connect/android/src/main/kotlin/network/particle/flutter/bridge/model/ChainData.java
package network.particle.flutter.bridge.model;

import androidx.annotation.Keep;

import com.google.gson.annotations.SerializedName;

@Keep
public class ChainData {

    @SerializedName("chain_id")
    public long chainId;

    @SerializedName("chain_id_name")
    public String chainIdName;

    @SerializedName("env")
    public String env;
}

File: particle-base/android/src/main/kotlin/network/particle/base_flutter/model/ChainData.kt
package network.particle.base_flutter.model


import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName

@Keep
class ChainData {
    @SerializedName("chain_name")
    var chainName: String? = null

    @SerializedName("chain_id")
    var chainId:Long = 0

    @SerializedName("env")
    var env: String? = null
}